ecms_base/features/custom/ecms_person/config/install/field.field.paragraph.person_additional_fields.field_person_field_label.yml

```diff
@@ -0,0 +1,28 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_person_field_label
+    - paragraphs.paragraphs_type.person_additional_fields
+    - taxonomy.vocabulary.person_additional_fields
+id: paragraph.person_additional_fields.field_person_field_label
+field_name: field_person_field_label
+entity_type: paragraph
+bundle: person_additional_fields
+label: 'Field label'
+description: ''
+required: true
+translatable: true
```

Comment:
I think this should be false, since this is an entity reference to the taxonomy term, which itself should be translatable.